custom_components/tuya_local/devices/morray_level_sensor.yaml

```diff
@@ -0,0 +1,109 @@
+name: Level sensor
+products:
+  - id: ywcgq
+    manufacturer: MORRAY
+    model: ME201WZS
+entities:
+  - entity: sensor
+    translation_key: status
+    class: enum
+    dps:
+      - id: 1
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: normal
+            value: normal
+          - dps_val: lower_alarm
+            value: empty
+          - dps_val: upper_alarm
+            value: full
```

Comment:
I think low and high are better here than empty and full, as the threshold is not 100 and 0, but whatever was set.